src/ansi-c/c_nondet_symbol_factory.cpp

```diff
@@ -51,6 +30,13 @@ class symbol_factoryt
   const bool assume_non_null;
   namespacet ns;
 
+  const symbolt &c_new_tmp_symbol(
```

Comment:
Why can't this use util/fresh_symbol?

Reply:
Yes, this should probably be replaced by a call to `get_fresh_aux_symbol()` since it's called in one place only.